blog/views.py

```diff
@@ -1,8 +1,19 @@
+"""Views for blog app.
+
+- post_list - returns post list;
+"""
+
 from django.shortcuts import render
 from .models import Post
 from django.utils import timezone
 # Create your views here.
 
+
 def post_list(request):
-    posts = Post.objects.filter(published_date__lte=timezone.now()).order_by('published_date')
-    return render(request, 'blog/post_list.html', {'posts' : posts})
+    """Show posts, which are published, ordered by published date."""
+    posts = (
+        Post.objects
```

Comment:
Yes, there are similar tools. Some of them are also supported by pre-commit http://pre-commit.com/hooks.html
- autopep8-wrapper
- autopep8
- yapf (adjusts code to follow Google's style guide, unlike once letting you follow PEP 8)